validation/src/main/scala/no/ndla/validation/TextValidator.scala

```diff
@@ -37,15 +35,20 @@ class TextValidator(allowHtml: Boolean) {
   def validate(
       fieldPath: String,
       text: String,
+      allowedTags: Set[_ <: String],
```

Comment:
Vi arver vel aldri fra string?
```suggestion
      allowedTags: Set[String],
```

Reply:
Nei, men måtte ha det for å kunne sende inn enten Set("") eller Set.empty. Kan sjå om eg kan endre det.

Reply:
`private val allowedTags = if (props.AllowHtmlInTitle) Set("span") else Set.empty` resolver til Set[_ <: String]

Reply:
Fant det! Måtte sette `Set.empty[String]`